src/components/Heading.jsx

```diff
@@ -0,0 +1,40 @@
+import { Box, Typography, createTheme, ThemeProvider } from "@mui/material";
+
+const fontTheme = createTheme({
```

Comment:
You should avoid creating new themes and using theme provider each time....

Ideally, only a single theme is required and this has been created in `theme-context.js` You are free to update the theme but creating new ones would give conflicts